LayoutTests/imported/w3c/web-platform-tests/wasm/serialization/module/broadcastchannel-success-and-failure-expected.txt

```diff
@@ -1,10 +1,5 @@
-CONSOLE MESSAGE: Unhandled Promise Rejection: DataCloneError: The object can not be cloned.
 
-Harness Error (FAIL), message = Unhandled rejection: The object can not be cloned.
-
-TIMEOUT WebAssembly.Module cannot cross agent clusters, BroadcastChannel edition Test timed out
-
-Harness Error (FAIL), message = Unhandled rejection: The object can not be cloned.
+Harness Error (TIMEOUT), message = null
```

Comment:
The failure tests cases are still failing. This is because they rely on the `messageerror` event which we don't support. When deserialization fails, we fire a regular `message` event  but with `data` being null.

Reply:
Do you plan on adding `messageerror`?

Reply:
Yes, we need to add it. I have a work-in-progress patch for it but haven't gotten around to finish it yet.

Reply:
I have verified that with the `messageerror` event implemented, this test and the one below start passing.
My patch to support the `messageerror` event is almost ready.